tool/tctl/common/plugin/awsic.go

```diff
@@ -0,0 +1,242 @@
+			Include: &types.AWSICResourceFilter_Id{Id: id},
+		})
+	}
+
+	return filters, nil
+}
+
+func (a *awsICArgs) parseUserFilters() ([]*types.AWSICUserSyncFilter, error) {
+	var result []*types.AWSICUserSyncFilter
+
+	if len(a.userOrigins) > 0 {
+		result = make([]*types.AWSICUserSyncFilter, 0, len(a.userOrigins))
+		for _, origin := range a.userOrigins {
+			result = append(result, &types.AWSICUserSyncFilter{
+				Labels: map[string]string{types.OriginLabel: origin},
+			})
+		}
+	}
+
+	if len(a.userLabels) > 0 {
+		result = append(make([]*types.AWSICUserSyncFilter, 0, len(result)+len(a.userLabels)), result...)
+		for _, labels := range a.userLabels {
+			result = append(result, &types.AWSICUserSyncFilter{Labels: labels})
+		}
+	}
+
+	return result, nil
+}
+
+func (p *PluginsCommand) initInstallAWSIC(parent *kingpin.CmdClause) {
+	p.install.awsIC.cmd = parent.Command("awsic", "Install an AWS IAM Identity Center integration.")
+	cmd := p.install.awsIC.cmd
+	cmd.Flag("access-list-default-owner", "Teleport user to set as default owner for the imported access lists. Multiple flags allowed.").
+		Required().
+		StringsVar(&p.install.awsIC.defaultOwners)
+	cmd.Flag("scim-url", "AWS Identity Center SCIM provisioning endpoint").
+		Required().
+		URLVar(&p.install.awsIC.scimURL)
+	cmd.Flag("scim-token", "AWS Identify Center SCIM provisioning token.").
+		Required().
+		StringVar(&p.install.awsIC.scimToken)
+	cmd.Flag("instance-region", "AWS Identity center instance region").
+		Required().
+		StringVar(&p.install.awsIC.region)
+	cmd.Flag("instance-arn", "AWS Identify center instance ARN").
+		Required().
+		StringVar(&p.install.awsIC.arn)
+	cmd.Flag("use-system-credentials", "Uses system credentials instead of OIDC.").
+		Default("true").
+		BoolVar(&p.install.awsIC.useSystemCredentials)
+
+	cmd.Flag("user-origin", fmt.Sprintf(`Shorthand for "--user-label %s=ORIGIN"`, types.OriginLabel)).
+		PlaceHolder("ORIGIN").
+		EnumsVar(&p.install.awsIC.userOrigins, types.OriginValues...)
+	cmd.Flag("user-label", "Add user label filter, in the form of a comma-separated list of \"name=value\" pairs. If no label filters are supplied, all Teleport users will be provisioned to Identity Center").
+		PlaceHolder("LABELSPEC").
+		SetValue(&p.install.awsIC.userLabels)
+
+	cmd.Flag("group-name", "Add AWS group to group import list by name. Can be a glob, or enclosed in ^$ to specify a regular expression. If no filters are supplied then all AWS groups will be imported.").
+		StringsVar(&p.install.awsIC.groupNameFilters)
+	cmd.Flag("account-name", "Add AWS Account to account import list by name. Can be a glob, or enclosed in ^$ to specify a regular expression. All AWS accounts will be imported if no items are added to account import list.").
+		StringsVar(&p.install.awsIC.accountNameFilters)
+	cmd.Flag("account-id", "Add AWS Account to account import list by ID. All AWS accounts will be imported if no items are added to account import list.").
```

Comment:
For these fields that are plural should we pluralize the flag name?

Reply:
I would read `--account-ids` as taking a list of account IDs, not each invocation of `--account-id` accepting one id each.